### PR TITLE
cluster alert guide

### PR DESCRIPTION
**Pull request by NBParis**
### What does this PR do?
Adds a guide to configure cluster alerts

### Motivation
It's pretty hacky but using min_cutoff and count_nonzero can get users what they want so adding a guide for them to look into a setup example was important. 

### Preview

https://docs-staging.datadoghq.com/nils/cluster-alert-guide/monitors/guide/create-cluster-alert/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/monitors/guide/create-cluster-alert.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+### Alert for a percentage of host with a high CPU
+
+If you don't want to be notified for each host that have a high CPU but only if a given percentage is in that situation, the overall idea is to leverage the `min_cutoff` and `count_nonzero` function available on metrics to do this. 
+
+For example, to get notified when 40% of hosts have a CPU above 50%, the idea is to split this in two parts:
+
+* Use the `min_cutoff` function to count the number of host that have a CPU above 50%
+* Use the `count_nonzero` function to count the total number of hosts
+* Divide one by the other to get the percentage of host with a high CPU
```
Comment (cswatt) [inline content/en/monitors/guide/create-cluster-alert.md:28]: In this example, you want to receive a notification when 40 percent of hosts have a CPU usage above 50 percent. Leverage the `min_cutoff` and `count_nonzero` functions:

* Use the `min_cutoff` function to count the number of hosts that have CPU usage above 50 percent.
* Use the `count_nonzero` function to count the total number of hosts.
* Divide one by the other for the resulting percentage of hosts with CPU usage above 50 percent.

File: content/en/monitors/guide/create-cluster-alert.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+{{< img src="monitors/faq/cluster-condition.png" alt="cluster-alert-condition"  >}}
+
+* Then set the condition to alert if the percentage of hosts in that condition reaches 40%
```
Comment (cswatt) [inline content/en/monitors/guide/create-cluster-alert.md:32]:
```suggestion
* Then, set the condition to alert if the percentage of hosts in that condition reaches 40 percent.
```

File: content/en/monitors/guide/create-cluster-alert.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+{{< img src="monitors/faq/cluster-trigger.png" alt="cluster-alert-trigger"  >}}
+
+This monitor then track the percentage of host that have a CPU above 50% within the last 10 minutes and generate a notification if more than 40% of those hosts meet the condition.
```
Comment (cswatt) [inline content/en/monitors/guide/create-cluster-alert.md:36]:
```suggestion
This monitor tracks the percentage of host that have a CPU usage above 50 percent within the last ten minutes and generates a notification if more than 40 percent of those hosts meet the specified condition.
```

**Review by cswatt — CHANGES_REQUESTED** [review]

**Comment by NBParis** [comment]
@ceciliawatt ready to merge when you are. Thanks for the updates

**Review by apigirl — APPROVED** [review]

**Pull request merged**